install-kubeadm.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/bash
+set -o nounset -o errexit
+
+CNI_VERSION="v0.6.0"
+mkdir -p /opt/cni/bin
+curl -L "https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-amd64-${CNI_VERSION}.tgz" | tar -C /opt/cni/bin -xz
+
+RELEASE="$(curl -sSL https://dl.k8s.io/release/stable.txt)"
```

Comment:
Question for @andrewsykim and @bhcleek: Do we want to bind to specific Kubernetes version? This will probably always install the latest version. In this case we need to be sure that CNI plugin will always work with the latest version, and other plugins such as CCM and CSI, or otherwise, the script can become broken.

Reply:
I'd prefer to bind to a specific version so we can update deliberately.